scenario_player/services/transactions/app.py

```diff
@@ -0,0 +1,19 @@
+from raiden.network.rpc.client import JSONRPCClient
+from scenario_player.services.common.factories import construct_flask_app
+from scenario_player.services.transactions.blueprints import (
+    transactions_blueprint, tokens_blueprint
+)
+
+from scenario_player.hooks.impl import HOOK_IMPL
+
+
+@HOOK_IMPL
+def register_blueprints():
+    """Register a list of blueprints with :mode:`raiden-scenario-player`."""
+    return [transactions_blueprint, tokens_blueprint]
```

Comment:
This hook implementation somehow is not called by the constructor on L17 below. I'm not sure where my spec/impl logic is faulty. Help is appreciated.

Reply:
Why should it be called? I don't see where the hooks are triggered.

Reply:
They're called here:
https://github.com/raiden-network/scenario-player/blob/7f5049e74a43a99393f61439125c9ff508bca952/scenario_player/hooks/__init__.py#L20

Reply:
Maybe I'm constructing the constant before the hook implementation can be registered?